source/org/zfin/db/postGmakePostloaddb/1152/db.changelog.master.xml

```diff
@@ -6,5 +6,6 @@
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <include file="source/org/zfin/db/postGmakePostloaddb/1152/ZFIN-8922.sql" />
+    <include file="source/org/zfin/db/postGmakePostloaddb/1152/ZFIN-8936.sql" />
```

Comment:
I assume this pertains to another ticket?

Reply:
It's for the same ticket. I forgot to commit it in the first PR.